src/BlazorWebFormsComponents/BoundField.razor.cs

```diff
@@ -2,24 +2,42 @@
 
 namespace BlazorWebFormsComponents
 {
-  /// <summary>
-  /// Bounds an object's property to a column by its property name 
-  /// </summary>
-  public partial class BoundField<ItemType> : BaseColumn<ItemType>
-  {
-
 	/// <summary>
-	/// Specifies the name of the object's property bound to the column
+	/// Binds an object's property to a column by its property name 
 	/// </summary>
-	[Parameter]
-	public string DataField { get; set; }
+	public partial class BoundField<ItemType> : BaseColumn<ItemType>
+	{
 
-	/// <summary>
-	/// Specifies which string format should be used.
-	/// </summary>
-	[Parameter]
-	public string DataFormatString { get; set; } = null;
+		/// <summary>
+		/// Specifies the name of the object's property bound to the column
+		/// </summary>
+		[Parameter]
+		public string DataField { get; set; }
+
+		/// <summary>
+		/// Specifies which string format should be used.
+		/// </summary>
+		[Parameter]
+		public string DataFormatString { get; set; }
 
+		public override RenderFragment Render(ItemType item)
+		{
+			var properties = DataField.Split('.');
+			object obj = item;
+			foreach (var property in properties)
+			{
 
-  }
+				var propertyInfo = obj.GetType().GetProperty(property);
```

Comment:
I think we need to check the property existence instead of TRUST user input

Reply:
This isn't end-user input but rather end-DEVELOPER input.. the folks who are using the component

Reply:
I do agree that there should be a better error. Right now this will null-ref (which is already did before - I didn't change it). I'll add some logic here.